shared/fileStorage.ts

```diff
@@ -56,4 +56,11 @@ export const getFilePath = (key: string) => {
   return `${vcDirectoryPath}/${key}.txt`;
 };
 
+export const getBackupFilePath = (key: string) => {
```

Comment:
lets keep backup & regular app-based storage operations in separate files.

cc @swatigoel Do you think this is a good idea?

Reply:
<img width="625" alt="Screenshot 2024-01-19 at 13 14 57" src="https://github.com/mosip/inji/assets/28667452/f3112a81-e47f-4a74-b4c0-2c6f75112244">
They are in seperate folders. Attaching Screenshot for reference